portal-api/jest.common.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  preset: 'ts-jest/presets/js-with-babel',
+  transform: {
+    '^.+\\.js$': 'babel-jest',
+    '^.+\\.ts$': 'ts-jest',
+  },
+};
```

Comment:
I _think_ because we use `preset: 'ts-jest/presets/js-with-babel',` here instead of just `preset: 'ts-jest',` then we may be able to remove the explicit `transform`?

Edit: just tried removing `transform` and all the unit tests still pass